src/components/DeleteModal/index.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import Modal from 'react-modal';
+import PropTypes from 'prop-types';
+import { Alert } from 'react-bootstrap';
+import { deleteArticle } from '../../redux/actions/articles';
+import Loader from '../Loader';
+
+import './modal.scss';
+
+const DeleteModal = (props) => {
+  const {
+    article, displayModal, closeModal, history
+  } = props;
+
+  const [state, setState] = useState({
+    loading: false,
+    errors: [],
+    successMessage: ''
+  });
+
+  const { errors, successMessage, loading } = state;
+
+  const handleDeleteArticle = async () => {
+    setState({ ...state, errors: [], loading: true });
+    const response = await props.deleteArticle(article);
+    const { success, errors: arrayOfErrors } = response;
+    if (arrayOfErrors) {
```

Comment:
@chingsley I have previously mentioned that the use of data types in variable names is not an acceptable convention.